reservation/tasks.py

```diff
@@ -0,0 +1,14 @@
+from .models import *
+
+
+def expired_reservations():
+    reservation = Reservation.objects.all()
+    for r in reservation:
```

Comment:
Here you make an update query to the db for each reservation that should be marked as expired. Can you write a single query that does a bulk update on all reservations that meet the conditions? It will be faster

Reply:
Am lasat un commit mai jos.